dataproc/instantiate_inline_workflow_template_test.go

```diff
@@ -28,14 +29,16 @@ func TestInstantiateInlineWorkflowTemplate(t *testing.T) {
 
 	region := "us-central1"
 
-	buf := new(bytes.Buffer)
+	testutil.Retry(t, 5, 60*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
 
-	if err := instantiateInlineWorkflowTemplate(buf, tc.ProjectID, region); err != nil {
-		t.Fatalf("instantiateInlineWorkflowTemplate got err: %v", err)
-	}
+		if err := instantiateInlineWorkflowTemplate(buf, tc.ProjectID, region); err != nil {
+			t.Errorf("instantiateInlineWorkflowTemplate got err: %v", err)
```

Comment:
Looks like this is still an issue. Should be `r.Errorf` followed by `return` rather than `t.Errorf`.

Using `r` makes it so the retry works.

Reply:
Fixed!